software/webdb.yml

```
@@ -0,0 +1,14 @@
+name: WebDB
+website_url: https://webdb.app
+source_code_url: https://gitlab.com/web-db/app
+description: Open Source and Efficient Database IDE.
+licenses:
+  - AGPL-3.0
+platforms:
+  - Docker
+tags:
+  - Database Management
+  - Software Development - IDE & Tools
+  - Self-hosting Solutions
+depends_3rdparty: false
+demo_url: https://demo.webdb.app/
```

This is optional so can be removed.
```suggestion
demo_url: https://demo.webdb.app/
```